site/_apps/nuxt-hn.txt

```diff
@@ -0,0 +1,29 @@
+---
+weight: 15
+title:  'HNPWA with Nuxt.js'
+github-title: 'nuxt/hackernews'
+libraries: 
+    - name: 'Vue'
+    - name: 'Nuxt'
+    - name: 'PWA module'
+module-bundling: 'Webpack'
+service-worker: 'Workbox'
+perfomance-patterns: 'PRPL, Prefetch/Preload JS + DNS + Data, Critical path CSS, Server side caching'
+server-side-rendering: 'Yes'
+api: 'Hacker News Firebase API'
+hosting: 'Now + CloudFlare CDN'
+authors: 
+    - name: 'Pooya'
+    - name: 'Sebastien'
+    - name: 'Alexandre'
+lighthouse: '91/100'
+interactive-em: '2.8s'
+interactive-faster-3g: '2.4s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170620_PG_a2a9feaf4ace07a61b2c6c2a171b1c79&run=1
+wpt-em-link: https://www.webpagetest.org/result/170620_AM_f5e1ae2f305c1d81b44cfe0c9c930e35
```

Comment:
EM_3G report finished. Main thread didn't settle down, but we did get a value for First Interactive at 4.5s.

![screen shot 2017-06-20 at 11 43 19 am](https://user-images.githubusercontent.com/61764/27349912-b9deec7a-55ad-11e7-9bf3-bab3d3332587.png)

https://www.webpagetest.org/result/170620_2Y_5f5fdbaae7d479adf52986a98fc8a720/
https://www.webpagetest.org/lighthouse.php?test=170620_2Y_5f5fdbaae7d479adf52986a98fc8a720&run=1